### DISCUSSION TITLE
Nesting Stacks vs. Copying nodes from a stack inside another - performance

### DISCUSSION THREAD

**Discussion by victoriomolina** [react-navigation/react-navigation#9450] [General]
Imagine 2 stack navigators "HomeStacks" and "SearchStacks". From the screens in HomeStacks, the user will be able to navigate to all screens in SearchStacks. In terms of performance, which is better? Option 1: Nesting SearchStacks inside HomeStacks ; Option 2: Copying all screens (nodes) of SearchStacks inside HomeStacks.

**Comment by cloudorbush** [comment]
I've personally found nesting to be worse for performance, in every case.
